package.json

```diff
@@ -226,6 +226,7 @@
     "pdfkit": "0.11.0",
     "pg": "8.1.0",
     "pg-hstore": "2.3.3",
+    "playwright": "^1.31.2",
```

Comment:
Fixed dep?

Reply:
I mean the dependency should not have a "^".

Though do we need this dependency? Or just upgrade @playwright/test? I notice the playwright docs don't install it manually.

Reply:
I set the version with the playwright image we are using. Good catch :)